### PR TITLE
Create html book store

### PR DESCRIPTION
**Pull request by ntimanu**
**In this branch my partner and I:**

- Added a function to add new book objects to populate the books array.
- Added a delete function to remove books.
- Added the local storage function.
- Added the README file

### CODE REVIEW

**Comment by prolajumokeoni** [comment]
Hi @Team

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have a descriptive pull request message and readme
Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: script.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  e.preventDefault();
+  addNewdata(title.value, author.value);
+});
+
+function removeBook(i) {
+  storedData.splice(i, 1);
+  updateData();
+  showBook();
+}
+removeBook();
```
Comment (AlphaNtihinduka) [inline script.js]: - [ ] I have noticed that when the page is reloaded, the first book in array is deleted. Kindly try as much as possible to ensure that on page reload there is no any book deleted in the local storage ☺️.

**Review by AlphaNtihinduka — CHANGES_REQUESTED** [review]
## Hi TEAM 🙋‍♂️🙋‍♂️
Good job working on implementing the awesome book functionality. You have done incredibly well. 👏  

TO HIGHLIGHT:
- Exact book removed ✔️ 
- Books stored in localstorage ✔️ 
- Books retrieved from localstorage ✔️ 
- Well-structured files :heavy_check_mark: 

# STATUS: Required Changes :recycle: 

close

## Required changes :recycle: 
- _Kindly consider the inline review comments ☺️_ 

## Optional suggestions
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or slack channel if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAPPY CODING AND GOOD LUCK :crossed_fingers: :four_leaf_clover:

**Review by danmainah — APPROVED** [review]
Hi @ntimanu and @Shabiblac , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights

✔️ Linter setup correctly and all checks are passing
✔️ Adding and Deleting a book functions working correctly

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**